### DISCUSSION TITLE
woorifisa-fe-tech-seminar/vue

### DISCUSSION THREAD

**Discussion by giscus** [Jiyun-Parkk/woorifisa-fe-tech-seminar#27] [General]
# woorifisa-fe-tech-seminar/vue

https://jiyun-parkk.github.io/woorifisa-fe-tech-seminar/vue

**Comment by devFancy** [comment]
깔끔하게 정리하셨네요!👍
덕분에 Vue.js를 왜 사용해야 하는지에 대해 이유를 알게됐어요!
잘 읽고 갑니다!😊

**Comment by Jiyun-Parkk** [comment]
재미있습니다

**Comment by ooutta** [comment]
Vue.js를 단순히 사용하는 것에만 그쳤는데, 
이번 기회를 통해서 왜 Vue.js를 사용해야 하는지,  이전 웹 개발에서는 어떤 방식을 사용하는지
여러 시연과 설명을 통해서 알게 되었습니다!
이러한 지식에 대해 상세히 전달해주셔서 감사합니다:)

**Comment by image00** [comment]
Vue를 사용하며 궁금했던 점이 해소되었습니다!

특히 반응성 부분이 어려웠는데, 예시자료를 통해 설명을 깔끔하게 잘 정리해주셨어요😊 
덕분에 이해했습니다! 좋은 자료 감사합니다:)
